crates/sui-mvr-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -484,6 +489,23 @@ impl<'a> LimitsTraversal<'a> {
         ))
     }
 
+    // If the field `f` is a multiGet query, extract the number of keys, otherwise return `None`.
+    // Returns an error if the number of keys cannot be represented as a `u32`.
+    fn multi_get_page_size(&mut self, f: &Positioned<Field>) -> ServerResult<Option<u32>> {
+        if !f.node.name.node.starts_with(MULTI_GET_PREFIX) {
+            return Ok(None);
+        }
+
+        let keys = f.node.get_argument(MULTI_GET_OBJECT_KEYS);
```

Comment:
The name of this constant seems to imply that it's specific to `multiGetObject` but it's not/shouldn't be -- all `multiGet`s now need to use `keys` as their parameter name for holding the keys.

```suggestion
        let keys = f.node.get_argument(MULTI_GET_KEYS);
```